skimage/util/dtype.py

```diff
@@ -252,8 +252,8 @@ def _convert(image, dtype, force_copy=False, uniform=False):
         return image
 
     if not (dtype_in in _supported_types and dtype_out in _supported_types):
-        raise ValueError("Can not convert from {} to {}."
-                         .format(dtypeobj_in, dtypeobj_out))
+        raise ValueError(f'Can not convert from {dtypeobj_in} to '
```

Comment:
```suggestion
        raise ValueError(f'Cannot convert from {dtypeobj_in} to '
```

unrelated to f-strings, but I think "cannot" is more common usage than "can not", although I think either is technically correct and both are more formal than `can't`

Reply:
Yes, [indeed](https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference). I had already made the [suggestion](https://github.com/scikit-image/scikit-image/pull/5508#discussion_r686933106) (and @MariosAchilias +1'd)), but it looks like it fell in the cracks in 774dc0c.